front/src/utils/trainId.ts

```diff
@@ -101,3 +101,6 @@ export const formatOccurrenceIdToEditoastTrainId = (occurrenceId: OccurrenceId):
 
   return formattedTrainId;
 };
+
+export const getPacedTrainIdFromOccurrenceId = (occurrenceId: OccurrenceId): PacedTrainId =>
```

Comment:
Can you write some tests for this function ? 
Also it would be nice to check that occurendId is valid

Reply:
there are already some, as I modified the existing function, I also modified the tests

Reply:
Hm not for `getPacedTrainIdFromOccurrenceId`. 
`formatPacedTrainIdToOccurrenceId` indeed has some tests (you modified it), but regarding `getPacedTrainIdFromOccurrenceId`, you created this function in this PR, but didn't add any tests

Reply:
oh yes, my bad those long name are not easy to differentiate sometimes.. done